lib/concepts/lesson/operations/create.rb

```diff
@@ -5,15 +5,25 @@ class Create
     class << self
       def call(lesson_params:)
         response = OperationResponse.new
-        response[:lesson] = Lesson.new(lesson_params)
+        response[:lesson] = build_lesson(lesson_params)
         response[:lesson].save!
         response
       rescue StandardError => e
         Sentry.capture_exception(e)
-        errors = response[:lesson].errors.full_messages.join(',')
-        response[:error] = "Error creating lesson: #{errors}"
+        response[:error] = "Error creating lesson: #{e}"
```

Comment:
I think this change might mean that you lose the validation error messages that were previously added to `lesson` if the record couldn't be saved. Without context I'm not sure if we need those on the front-end or not, but perhaps you could merge the validation errors from Lesson and Project and return those?

Reply:
Ah, yes, you're right, I will revert - thanks!